src/foam/core/AbstractMapPropertyInfo.java

```diff
@@ -34,8 +36,10 @@ public int compare(Object o1, Object o2) {
       if ( v1 == null && v2 == null ) continue;
       if ( v2 == null ) return 1;
       if ( v1 == null ) return -1;
-      if ( v1 instanceof Comparable) 
+      if ( v1 instanceof Comparable)
```

Comment:
space before )

Reply:
Fixed in https://github.com/kgrgreer/foam3/pull/85/commits/ed8dca623e7fec2016db04e92caa978026bd5750.